benchmarks/gemm/gemm_configuration_sycl.hpp

```diff
@@ -149,6 +149,87 @@ struct GemmConfiguration<
   }
 };
 
+/////////////////////////////////////////////////////////////////////////
+
+// float16
+
+template<typename LayoutA, typename LayoutB, typename LayoutC,
+  class TileShape, class TiledMma, class GmemTiledCopyA, class GmemTiledCopyB, Scheduler TileScheduler, class EpilogueOp>
+struct GemmConfiguration<
```

Comment:
The only difference with the above specialization is `cutlass::bfloat16_t` -> `cutlass::half_t`. I think you should just add another template argument (or two) to to the above specialization to capture `ElementA` and `ElementB` as template parameters.
You can add a check like below.
```
static_assert(cute::is_any_of<ElementA, cutlass::bfloat16_t, cutlass::half_t>, "only bfloat and half types are supported for A currently");
static_assert(cute::is_same<ElementA, ElementB>);
```